### PR TITLE
AG-7263 - Fix area series missing Y-value property

### PR DESCRIPTION
**Pull request by alantreadway**
https://ag-grid.atlassian.net/browse/AG-7263

Fixes a regression from `27.2.0` where a missing-Y value property leads to a sub-stack being offset.

### CODE REVIEW

File: charts-packages/ag-charts-community/src/chart/series/cartesian/areaSeries.test.ts [hunk 1]
```diff
@@ -90,10 +94,6 @@ describe('AreaSeries', () => {
             console.warn = jest.fn();
         });
 
-        afterEach(() => {
-            expect(console.warn).not.toBeCalled();
```
Comment (manapeirov) [inline charts-packages/ag-charts-community/src/chart/series/cartesian/areaSeries.test.ts:94]: what warning does it give for this specific example?

**Review by manapeirov — APPROVED** [review]

File: charts-packages/ag-charts-community/src/chart/series/cartesian/areaSeries.test.ts (hunk 1 above)
Comment (alantreadway) [reply charts-packages/ag-charts-community/src/chart/series/cartesian/areaSeries.test.ts:94]: `ag-charts-community.min.js:11 The key 'safari' was not found in the data: [Object]`

I'm undecided on whether this warning is necessary tbh

Comment (alantreadway) [reply charts-packages/ag-charts-community/src/chart/series/cartesian/areaSeries.test.ts]: After some thought, I've decided this message is unnecessary - I've removed it and reverted the test change - PTAL!

**Review by manapeirov — APPROVED** [review]
Looks great thanks Alan

**Pull request merged**